exercises/README.md

```diff
@@ -4,5 +4,6 @@
 * [Exercise #2](./2/README.md)
 * [Exercise #3](./3/README.md)
 * [Exercise #4](./4/README.md)
+* [Exercise #5](./5/README.md)
 
```

Comment:
I think this should not appear here when you rebase on the homework-repo but im not sure...